src/Services/AppointmentService.php

```diff
@@ -33,6 +33,14 @@ class AppointmentService extends BaseService
     const PRACTITIONER_TABLE = "users";
```

Comment:
This class has so many code lines, It would be a very good idea to separate some things into other classes, for example validation logic, rendering logic, etc.

Validation.php
Render.php